schema.sql

```diff
@@ -1,11 +1,23 @@
-/* Database schema to keep the structure of entire database. */
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(100),
+  age INT
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  species_name VARCHAR(100)
+);
 
 CREATE TABLE animals (
```

Comment:
- [OPTIONAL] Here your were supposed to alter an existing table not create a new. Some like so: 
```
   ALTER TABLE REMOVE species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```
Please kindly take not of that.